### PR TITLE
Colorize buildpack failures

### PR DESCRIPTION
**Pull request by schneems**
Example output for Errors:

Example output for Warnings:

### CODE REVIEW

File: lib/language_pack/base.rb [hunk 1]
```diff
@@ -91,6 +91,7 @@ def compile
         topic "DEPRECATIONS:"
         puts @deprecations.join("\n")
       end
+      Kernel.puts "\e[0m"
```
Comment (sigmavirus24) [inline lib/language_pack/base.rb]: Do we want a comment to indicate this is "default"?

File: lib/language_pack/shell_helpers.rb [hunk 2]
```diff
@@ -62,6 +62,22 @@ def self.initialize_env(path)
...
 
+    # Should only be used once in the top level script
+    # for example $ bin/support/ruby_compile
+    def self.display_error_and_exit(e)
+      Kernel.puts "\e[1m\e[31m" # Bold Red
+      Kernel.puts " !"
+      e.message.split("\n").each do |line|
+        Kernel.puts " !     #{line.strip}"
+      end
+      Kernel.puts " !\e[0m"
```
Comment (sigmavirus24) [inline lib/language_pack/shell_helpers.rb]: Same question here

**Comment by schneems** [comment]
Worth mentioning that I tested this on windows 8 and it works

**Comment by schneems** [comment]
@dmathieu is there a reason you deleted the branch?

**Pull request merged**